<?xml version="1.0"?>
<launch>

    <include file="$(find probot_gazebo)/launch/probot_anno_with_gripper/probot_anno_with_gripper_bringup_moveit.launch" />

    <node pkg="probot_grasping" name="graspingDemo" type="graspingDemo" output="screen">
        <param name="table_length"          value="0.3"/>
        <param name="table_breadth"         value="0.3"/>
        <param name="pregrasp_x"            value="0.20"/>
        <param name="pregrasp_y"            value="-0.17"/>
        <param name="pregrasp_z"            value="0.28"/>
         <param name="grasp_x"            value="0.12"/>
         <param name="grasp_y"            value="0.3"/>
         <param name="hor_grasp_z"            value="-0.0055"/>
         <param name="ver_grasp_z"            value="-0.001"/>
         <param name="hor_ratio"            value="0.4"/>
         <param name="ver_ratio"            value="0.4"/>
         <param name="change_angle"            value="0"/>
        <param name="real_z"            value="0.15"/>
        <param name="simulation"            value="1"/>
        
         <!-- 0.06 -0.012-->
    </node>

</launch>